content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -10,25 +10,30 @@ test: no
 ---
 
 A **waypoint proxy** is an optional deployment of the Envoy-based proxy to add Layer 7 (L7) processing to a defined set of workloads.
-Waypoint proxies are installed, upgraded and scaled independently from applications; an application owner should be unaware of their existence. Compared to the sidecar data plane mode, which runs an instance of the Envoy proxy alongside each workload, the number of proxies required can be substantially reduced.
+
+Waypoint proxies are installed, upgraded and scaled independently from applications; an application owner should be unaware of their existence. Compared to the sidecar {{< gloss >}}data plane{{< /gloss >}} mode, which runs an instance of the Envoy proxy alongside each workload, the number of proxies required can be substantially reduced.
 
 A waypoint, or set of waypoints, can be shared between applications with a similar security boundary. This might be all the instances of a particular workload, or all the workloads in a namespace.
 
+As opposed to {{< gloss >}}sidecar{{< /gloss >}} mode, in ambient mode policies are enforced by the **destination** waypoint. In many ways, the waypoint acts as a gateway into the namespace. Istio enforces that all traffic coming into the namespace goes through the waypoint, which then enforces all policies for that namespace.
```

Comment:
Would we then be best to change the next line about "coming into the namespace" etc, as it's not just about a namespace, it can also be for a service/workload?

Reply:
I don't think I see that line but yeah, basically anywhere we imply that a waypoint is like ingress for a namespace I'd vote to reword that.